### ISSUE TITLE
Find new name for the project

### ISSUE THREAD

**Issue by olivershooter** [olivershooter/plog#15]
Working name is PLOG, but we need a cooler name.

**Comment by olivershooter** [comment]
Copying from the old Trello:
Spewing whatever comes to me:

    Fax
    Paperpie
    Chatterly
    Quiller
    Penor
    Swritten
    Ink Link
    Nook

Looking into Nook

[Nook.com](http://nook.com/):
owned by (now out of business) Barnes and Noble
Expires 2026

[Nook.net](http://nook.net/)
Also owned by Barnes and Noble
Expires November

[Nook.io](http://nook.io/)
Some invoice payment tool

Chances are low [Nook.net](http://nook.net/) wont be snatched up so I looked into others and discovered .book

[What could have been.](https://webmasters.stackexchange.com/questions/121909/status-of-the-book-tld)

TL:DR - we will never get the nook.book domain and fuck amazon

Nook.works is £160 on namecheap
Nook.surf is £16
nook.institute £5
nook.academy £12

[Nook.com](http://xn--12Nook-9ga86043ca.com) is a no brainer imo but the others not so much.
